src/main/scala/za/co/absa/abris/avro/registry/ConfluentRegistryClient.scala

```diff
@@ -23,6 +24,10 @@ import scala.collection.JavaConverters._
 class ConfluentRegistryClient(client: SchemaRegistryClient) extends AbstractConfluentRegistryClient(client) {
 
   def this(configs: Map[String,String]) = this(ConfluentRegistryClient.createClient(configs))
+
+  def this(restService: RestService, maxSchemaObject: Int) = {
+    this(ConfluentRegistryClient.createClient(restService, maxSchemaObject))
+  }
 }
```

Comment:
1. Lets update the files where this class is being used.
2. Add Test cases
3. Add scaladoc for this in terms of how to use this constructor